pester/Pester
# Add more assertion

changelog: 
- Add -Because parameters to all assertions Fixes #312
- Add -BeLessOrEqual and -BeGreaterOrEqual
- Add -Contain (that operates on arrays) Fixes #121
- Add -BeLikeExactly
- Add -HaveType alias to -BeOfType 
- Fix assertion messages in -BeOfType Fixes #729
- Throw argument exception when -BeOfType is given type that is not loaded
- Add -PassThru to -Throw to get the exception when some is thrown and passes the filters
- Add -BeTrue to test for truthy values 
- Add -BeFalse to test for falsy values
- Add -HaveCount to count stuff in collections

Remove many duplicated tests in Be, and rewrite many tests using Verify to remove interdependencies between Shoulds
